lib/internal/modules/cjs/loader.js

```diff
@@ -423,15 +423,16 @@ function tryPackage(requestPath, exts, isMain, originalPath) {
       err.code = 'MODULE_NOT_FOUND';
       err.path = path.resolve(requestPath, 'package.json');
       err.requestPath = originalPath;
-      // TODO(BridgeAR): Add the requireStack as well.
+
       throw err;
     } else {
       const jsonPath = path.resolve(requestPath, 'package.json');
       process.emitWarning(
         `Invalid 'main' field in '${jsonPath}' of '${pkg}'. ` +
           'Please either fix that or report it to the module author',
         'DeprecationWarning',
-        'DEP0128'
+        'DEP0128',
+        { requireStack }
```

Comment:
can we generate the requireStack in this way before using it here https://github.com/nodejs/node/blob/7811d2de699d31e76b8ff9b251df68f7706b2460/lib/internal/modules/cjs/loader.js#L1027-L1032? the variable is currently undefined